src/test/java/com/epam/jamp2/model/CalculationCommandTest.java

```diff
@@ -0,0 +1,85 @@
+package com.epam.jamp2.model;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import org.junit.Assert;
+import org.junit.Test;
+import static org.hamcrest.Matchers.*;
+
+//@RunWith(SpringJUnit4ClassRunner.class)
+public class CalculationCommandTest {
+
+	@Test
+	public void testCommand1() throws CommandFormatException {

```

Comment:
Try using more meaningful names for your test, it will simplify investigation process when tests start failing for some reason.